src/nile/cli.py

```diff
@@ -120,10 +140,21 @@ def setup(signer, network):
 @click.argument("method", nargs=1)
 @click.argument("params", nargs=-1)
 @click.option("--max_fee", nargs=1)
+@network_option
+@click.option("--track", "-t", "watch_mode", flag_value="track")
```

Comment:
Should we compress both pairs (watch_mode and query) into a single option (like network_option) to reduce unnecessary verbosity in the code?

Reply:
I think it makes more sense to do so with `watch_mode` as opposed to `query` because we only insert `query` into `account.send`. I'm happy to add it if think otherwise.